index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
Kindly include a "<title>" tag inside your "head" tag, as your website does not have a title name at the moment.

Reply:
@Ofarouq310, Great job spotting the issue 👍 , however, it is good to let the student know `WHY` this change is necessary